content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -22,6 +22,17 @@ Kubernetes, on Linux:
 - [CRI-O](#cri-o)
 - [Docker](#docker)
 
+{{< note >}}
+Dockershim, the module that allowed Kubernetes to interact with Docker has been [deprecated](
```

Comment:
Rather than “has been”, I recommend “was”. The reason is that “has been” suggests it's a recent removal and we prefer the text of documentation to be more timeless (and easier to maintain).

Reply:
@mikebrow  @sftim done.